api/Console.json

```diff
@@ -586,7 +586,7 @@
       "exception": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Console/exception",
-          "description": "Alias for Console.error",
+          "description": "<code>Console.exception</code> alias for <code>Console.error</code>",
```

Comment:
Add 'is an'?

Reply:
Good idea! Though on second look, I decided it should probably look a bit more like the other feature, not adding superfluous "Console" text. Thanks!